demisto_sdk/commands/common/tests/pack_unique_files_test.py

```diff
@@ -550,3 +550,25 @@ def test_validate_pack_readme_and_pack_description_no_readme_file(self, repo):
             assert '"README.md" file does not exist, create one in the root of the pack' in self.validator.get_errors()
             assert 'README.md content is equal to pack description. ' \
                    'Please remove the duplicate description from README.md file' not in self.validator.get_errors()
+
+    @pytest.mark.parametrize('pack_description, is_valid', [
+        ('Hey there, just testing', True),
+        ('This is will fail cause the description here is too long.test test test test test test test test test test'
+         'test test test test test test test test test test test test', True),
+    ])
+    def test_is_pack_metadata_desc_too_long(self, repo, pack_description, is_valid):
+        """
+        Given:
+            - Case A: Valid description length
+            - Case B: Invalid description length - higher than 130
+
+        When:
+            - Validating pack description length
+
+        Then:
+            - Case A: Ensure validation passes as the description field length is valid.
+            - Case B: Ensure validation passes although description field length is higher than 130-
+            will print a warning.
+        """
+
+        assert self.validator.is_pack_metadata_desc_too_long(pack_description) == is_valid
```

Comment:
let's ensure that the correct warning is also printed or at least passed to the _add_error.